screens/About.js

```diff
@@ -8,35 +8,53 @@ import {
   ScrollView,
   TouchableHighlight,
   StatusBar,
-  Platform
+  Platform,
 } from "react-native";
+import PropTypes from "prop-types";
 import { Header } from "react-native-elements";
 import Icon from "react-native-vector-icons/MaterialIcons";
-import GLOBAL from "../utils/globals";
 import { connect } from "react-redux";
 import VersionNumber from "react-native-version-number";
+import GLOBAL from "../utils/globals";
 import AnalyticsManager from "../utils/analytics";
 import Strings from "../utils/localization";
 
 class Home extends React.Component {
   componentDidMount() {
-    AnalyticsManager.getInstance().trackScreenView(
-      "About",
-      this.constructor.name
-    );
+    AnalyticsManager.getInstance().trackScreenView("About", this.constructor.name);
   }
 
   render() {
+    const styles = StyleSheet.create({
+      container: {
+        flex: 1,
+        padding: 8,
+        backgroundColor: "#fff",
```

Comment:
Veerji, do we have colors in a common file? If yes, you can use that file variable here otherwise you can create a new one.

Reply:
fixed